table.csv

```diff
@@ -35,6 +35,8 @@ multiaddr,          ,                         0x32
 multibase,          ,                         0x33
 
 multihashes,,
+md4,                ,                         0xd3
+md5,                ,                         0xd4
```

Comment:
Well, if we are going to arbitrarily pick a number, better make it `0xd4` and `0xd5`. Sounds good @Kubuxu ?

Reply:
SGWM

Reply:
Done and done.